d2cli/watch.go

```diff
@@ -430,15 +432,23 @@ func (w *watcher) handleRoot(hw http.ResponseWriter, r *http.Request) {
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>%s</title>
-	<script src="./static/watch.js"></script>
-	<link rel="stylesheet" href="./static/watch.css">
-	<link id="favicon" rel="icon" href="./static/favicon.ico">
+	<script src="/static/watch.js"></script>
+	<link rel="stylesheet" href="/static/watch.css">
+	<link id="favicon" rel="icon" href="/static/favicon.ico">
 </head>
 <body data-d2-dev-mode=%t>
 	<div id="d2-err" style="display: none"></div>
 	<div id="d2-svg-container"></div>
 </body>
 </html>`, filepath.Base(w.outputPath), w.devMode)
+
+	// if path is "/x.svg", we just want "x"
+	split := strings.Split(strings.TrimPrefix(r.URL.Path, "/"), ".")
```

Comment:
Use `strings.IndexLastByte` here instead to clearly indicate what the code is trying to do here.